WordPress/Classes/ViewRelated/Media/MediaLibraryViewController.swift

```diff
@@ -123,6 +123,8 @@ class MediaLibraryViewController: WPMediaPickerViewController {
             navigationItem.setLeftBarButton(UIBarButtonItem(barButtonSystemItem: .cancel, target: self, action: #selector(editTapped)), animated: false)
 
             let trashButton = UIBarButtonItem(image: .gridicon(.trash), style: .plain, target: self, action: #selector(trashTapped))
+            trashButton.accessibilityLabel = NSLocalizedString("Trash", comment: "Accessibility label for trash button in nav bar")
```

Comment:
I would probably clarify these comments slightly by saying `for trash button to delete items from the user's media library`

Reply:
Similar goes for the add buttons below.

Reply:
Np done!